Makefile

```diff
@@ -33,16 +33,16 @@ clean:
 	$(RM) -R build/
 
 # All files which the build depends on
-MD_FILES=$(shell find -path "./src/*" -name "*.md")
-RST_FILES=$(shell find -path "./src/*" -name "*.rst")
-PNG_FILES=$(shell find -path "./src/*" -name "*.png")
+MD_FILES=$(shell find ./src -name "*.md")
+RST_FILES=$(shell find ./src -name "*.rst")
+PNG_FILES=$(shell find ./src -name "*.png")
```

Comment:
Two concerns I have, here:
1. Might still be good to quote the paths.
2. Are there going to be any side effects to omitting the wildcard on the `src/*` path?

Reply:
1. Is totally gratuitous since there are no variable expansions, wildcards, or anything going on, but whatever floats your boat (I committed this change with an incorrectly formatted commit message, then tried to fix it and made a mess of the history, oops)
2. Nope, I don't think so.  `find -path "./src/*"` says "search in `.` for paths matching `./src/*`" while `find "./src/"` says "search in `./src/`", so the latter would avoid even searching in other directories in `.` other than `src`.